dev/bots/pubspec.yaml

```diff
@@ -23,9 +25,7 @@ dev_dependencies:
   file: 2.3.7 # TRANSITIVE DEPENDENCY
   glob: 1.1.5 # TRANSITIVE DEPENDENCY
   html: 0.13.2+2 # TRANSITIVE DEPENDENCY
-  http: 0.11.3+16 # TRANSITIVE DEPENDENCY
   http_multi_server: 2.0.4 # TRANSITIVE DEPENDENCY
-  http_parser: 3.1.1 # TRANSITIVE DEPENDENCY
```

Comment:
please make sure to run `flutter update-packages --force-upgrade` to update the `pubspec.yaml` files.